docs/adopters.md

```diff
@@ -177,5 +177,5 @@ G8 templates provide a fast way to get started with SBT projects by just running
 ----
 
 @:style(footer)
-[@:icon(edit) Add yourself](https://github.com/http4s/http4s/edit/main/website/src/hugo/content/adopters.md), alphabetically, if you please.
+[@:icon(edit) Add yourself](https://github.com/http4s/http4s/edit/main/docs/adopters.md), alphabetically, if you please.
```

Comment:
Nice catch! Just a little nitpick from me: this PR targets the `series/0.23` branch, but the link contains `main`. I believe we can fix that for `0.23`, as well as for `1.0`.

Reply:
The PR targets `series/0.23` because that is the default branch. It wasn't a conscious decision from me. :-)

Should it target `main` instead or should the branch in the URL changed to `series/0.23`?

Reply:
The PR can target main actually, that is an unversioned page that is only published from the main branch.

Thanks btw! :)

Reply:
It targets main now.